application-workloads/darktrace-vsensor-standalone/azuredeploy.json

```diff
@@ -0,0 +1,291 @@
+    "contentVersion": "1.0.0.5",
+    "parameters": {
+        "location": {
+            "defaultValue": "[resourceGroup().location]",
+            "type": "String",
+            "metadata": {
+                "description": "Location for all resources. The default is the Resource Group location."
+            }
+        },
+        "virtualNetworkResourceGroupName": {
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Virtual Network Resource Group."
+            }
+        },
+        "virtualNetworkName": {
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Virtual Network, should be in the above Location."
+            }
+        },
+         "subnet1Name": {
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Subnet in the Virtual Network."
+            }
+        },
+        "networkSecurityGroupName": {
+            "type": "String",
+            "metadata": {
+                "description": "Name for the Network Security Group that the template will create. Note that a pre-existing Network Security Group with the same name will be replaced."
+            }
+        },
+        "MgmtSourceAddressOrRange": {
+            "type": "String",
+            "metadata": {
+                "description": "Provide an address range using CIDR notation (e.g. 10.1.0.0/24), or an IP address (e.g. 192.168.99.21) for Management access via ssh (port 22/TCP). You can also provide a comma-separated list of IP addresses and/or address ranges (a valid comma-separated list is 10.1.0.4,10.2.1.0/24). To allow access from any IP you can use 0.0.0.0/0."
+            }
+        },
+        "vsensorName": {
+            "type": "String",
+            "metadata": {
+                "description": "The name of the vSensor."
+            }
+        },
+        "numberOfVsensors": {
+            "type": "Int",
+            "defaultValue": 1,
+            "minValue": 1,
+            "maxValue": 4,
+             "metadata": {
+                "description": "Number of vSensors to be deployed, max is 4."
+            }
+        },
+        "diskSize": {
+            "defaultValue": 30,
+            "minValue": 30,
+            "maxValue": 1024,
+            "type": "Int",
+            "metadata": {
+                "description": "The vSensor disk size in GB between 30 and 1024. Check the Darktrace customer portal for more information."
+            }
+        },
+        "virtualMachineSize": {
+            "defaultValue": "Standard_D2s_v3",
+            "type": "String",
+            "metadata": {
+                "description": "The VM size. Check the Darktrace customer portal for more information about the vSensor Virtual Hardware requirements."
+            }
+        },
+        "adminUsername": {
+            "defaultValue": "ubuntu",
```

Comment:
remove defaultValue (increases surface area)

Reply:
defaultValue removed.

May I ask what 'increases surface area' means in this context?

Reply:
Users tend to use defaultValues and having a username be known to an attacker gives them more to work with...